Alien.cpp

```diff
@@ -1,10 +1,11 @@
 #include "Alien.h"
+#include <iostream>
 
 void foreign_life::Alien::set_type(int type) {
     this->type = type;
 }
 
-void foreign_life::Alien::print_type() {
+void foreign_life::Alien::print_type() const {
     std::cout << "Type of alien is " << this->type
          << std::endl;
-}
\ No newline at end of file
+}
```

Comment:
You can configure vscode to always make sure the file ends with a newline character